content/blog/how-to-create-a-virtual-network-in-hpe-greenlake-for-private-cloud.md

```diff
@@ -0,0 +1,170 @@
+---
+title: How to create a virtual network in HPE GreenLake for Private Cloud Enterprise
+date: 2022-06-07T05:34:46.180Z
+featuredBlog: false
+author: Vinnarasu Ganesan, Thavamaniraja S
+authorimage: /img/Avatar1.svg
+thumbnailimage: /img/greenlake-576x324.jpg
+tags:
+  - hpe-greenlake
+---
+## Introduction
+
+HPE GreenLake for Private Cloud Enterprise is: designed to deliver and help manage a private cloud. Available on the HPE GreenLake Central platform, the HPE GreenLake for Private Cloud Enterprise is
+
+* An HPE designed, implemented, owned, and operated private cloud that is deployed at a customer site
+* Offered as a consumption-based service that enables customers to better align costs to outcomes
+* An intuitive self-service portal UI used to create and manage private cloud services such as compute, storage, and network (example described in this blog)
+
+This blog post explains the steps to create a virtual network with a static IP pool and DHCP using NSX-T, network virtualization, and a security platform that enables the virtual cloud network in HPE GreenLake for private cloud.
+
+## Understanding private cloud networking
+
+The following illustration shows how you can use NSX objects to achieve NSX logical networking in HPE GreenLake for Private Cloud Enterprise.
+
+![](/img/fig-2.jpg)
+
+The network components are as follows:
+
+##### **VM**
```

Comment:
You may want to revisit the font size for these network components in this section as per Dale's suggestion. Maybe you can try heading level 4 or level 3, or a standard text in bold.

Reply:
Keeping as standard text in Bold. Hope that is fine.